### ISSUE TITLE
ci/circleci: e2e-pilot — Your tests failed on CircleCI

### ISSUE THREAD

**Issue by mangchiandjjoe** [istio/istio#3424]
It started happening today. 

https://circleci.com/gh/cf-routing/istio/3127?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
=================== Failed tests! =====================
6 errors occurred:

* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 0 expected 100 requests (+/-5) to reach v1 => Got 50
* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 0 expected 0 requests (+/-5) to reach v2 => Got 50
* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 1 expected 100 requests (+/-5) to reach v1 => Got 51
* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 1 expected 0 requests (+/-5) to reach v2 => Got 49
* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 2 expected 0 requests (+/-5) to reach v2 => Got 50
* routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 2 expected 100 requests (+/-5) to reach v1 => Got 50


2018-02-12T23:41:26.531402Z	info	

=================== Skipping teardown =====================
(auth infra)


2018-02-12T23:41:26.531457Z	error	

=================== Failed infrastructure tests! =====================
6 errors occurred:

* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 0 expected 100 requests (+/-5) to reach v1 => Got 50
* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 0 expected 0 requests (+/-5) to reach v2 => Got 50
* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 1 expected 100 requests (+/-5) to reach v1 => Got 51
* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 1 expected 0 requests (+/-5) to reach v2 => Got 49
* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 2 expected 0 requests (+/-5) to reach v2 => Got 50
* (auth infra) routing-rules run 0 routing all traffic to c-v1 with shadow policy attempt 2 expected 100 requests (+/-5) to reach v1 => Got 50


make: *** [e2e_pilot] Error 255
Exited with code 2
```

**Comment by mangchiandjjoe** [comment]
CC @wattli

**Comment by mangchiandjjoe** [comment]
@andraxylia Could you please take a look or reassign the issue to the right person?

**Comment by mangchiandjjoe** [comment]
@rshriram It seems that you can take a look at this issue. Could you please take a look or assign to the right person?

**Comment by rosenhouse** [comment]
I think y'all are aware, but that link up top is to a fork (specifically, the Cloud Foundry team's fork branch for https://github.com/istio/istio/pull/3412 )

**Comment by rshriram** [comment]
This has been fixed (#3442 )
